src/main/js/doc.js

```diff
@@ -626,7 +626,7 @@
 
         /* resolve desired timing for regions */
 
-        for (var region_i in doc.head.layout.regions) {
+        for (var region_i = 0; region_i < doc.head.layout.regions.length; region_i++) {
```

Comment:
`tt.head.layout.regions` is a dictionary, so this should in fact use `for..in`

Reply:
hi @palemieux sorry for the delay. Thanks for the review. I'll address your comments very soon.